src/engine/controls/bpmcontrol.h

```diff
@@ -23,7 +24,7 @@ class BpmControl : public EngineControl {
 
   public:
     BpmControl(QString group, UserSettingsPointer pConfig);
-    ~BpmControl() override;
+    ~BpmControl() override = default;
```

Comment:
This change probably produces the build error with scons on Travis.
You may try to revert it.